blueprints/grafana/creator.libsonnet

```diff
@@ -47,5 +62,9 @@ function(policyFile, cfg) {
   local panels = flowControlPanels + otherPanels,
   local final = dashboard.baseDashboard
                 + g.dashboard.withPanels(panels),
+
+  local receiverDashboard = dashboard.baseDashboard
+                            + g.dashboard.withPanels(receiverPanels),
   dashboard: final,
+  additionalDashboard: receiverDashboard,
```

Comment:
Wrong assumption. There can be multiple additional dashboards once you change infra meter parsing code to a loop